README.md

```diff
@@ -21,36 +21,28 @@ conflicts needed to be manually resolved for each upgrade merge. To minimize the
 
 ## Usage
 
-Install a [Cookiecutter](https://github.com/audreyr/cookiecutter)
-template on current directory:
+Install a [Cookiecutter](https://github.com/audreyr/cookiecutter) template:
```

Comment:
shouldn't this link to our gitlab archetype?

Reply:
Nope this is a generic tool built on top of the `cookiecutter` project itself so it's actually unaware of any of Zillow's archetypes :D